[python example] sqlite error

<pre>
vagrant@vagrant-xen-trusty64:~/rumprun-packages/python$ x86_64-rumprun-netbsd-gcc examples/hw.c \
>     -o examples/hw \
>     -Ibuild/pythondist/include/python3.4m \
>     -Lbuild/pythondist/lib \
>     -lpython3.4m -lutil -lm -lz -lssl -lcrypto
/home/vagrant/rumprun/rumprun/lib/libcrypto.a(e_rc5.o): In function `rc5_32_12_16_cbc_cipher':
/home/vagrant/rumprun/src-netbsd/crypto/external/bsd/openssl/dist/crypto/evp/e_rc5.c:80: warning: RC5 is a patented algorithm; link against libcrypto_rc5
/home/vagrant/rumprun/rumprun/lib/libcrypto.a(rc5ofb64.o): In function `RC5_32_ofb64_encrypt':
/home/vagrant/rumprun/src-netbsd/crypto/external/bsd/openssl/dist/crypto/rc5/rc5ofb64.c:90: warning: RC5 is a patented algorithm; link against libcrypto_rc5
/home/vagrant/rumprun/rumprun/lib/libcrypto.a(rc5_ecb.o): In function `RC5_32_ecb_encrypt':
/home/vagrant/rumprun/src-netbsd/crypto/external/bsd/openssl/dist/crypto/rc5/rc5_ecb.c:77: warning: RC5 is a patented algorithm; link against libcrypto_rc5
/home/vagrant/rumprun/rumprun/lib/libcrypto.a(e_rc5.o): In function `r_32_12_16_init_key':
/home/vagrant/rumprun/src-netbsd/crypto/external/bsd/openssl/dist/crypto/evp/e_rc5.c:117: warning: RC5 is a patented algorithm; link against libcrypto_rc5
examples/hw: In function `module_complete':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/module.c:107: undefined reference to `sqlite3_complete'
examples/hw: In function `module_enable_shared_cache':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/module.c:136: undefined reference to `sqlite3_enable_shared_cache'
examples/hw: In function `PyInit__sqlite3':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/module.c:442: undefined reference to `sqlite3_libversion'
examples/hw: In function `pysqlite_statement_dealloc':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:393: undefined reference to `sqlite3_finalize'
examples/hw: In function `pysqlite_statement_create':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:76: undefined reference to `sqlite3_prepare'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:86: undefined reference to `sqlite3_finalize'
examples/hw: In function `pysqlite_statement_bind_parameter':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:130: undefined reference to `sqlite3_bind_int64'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:159: undefined reference to `sqlite3_bind_blob'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:134: undefined reference to `sqlite3_bind_double'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:145: undefined reference to `sqlite3_bind_text'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:102: undefined reference to `sqlite3_bind_null'
examples/hw: In function `pysqlite_statement_bind_parameters':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:197: undefined reference to `sqlite3_bind_parameter_count'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:256: undefined reference to `sqlite3_bind_parameter_name'
examples/hw: In function `pysqlite_statement_recompile':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:317: undefined reference to `sqlite3_prepare'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:333: undefined reference to `sqlite3_bind_parameter_count'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:334: undefined reference to `sqlite3_transfer_bindings'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:341: undefined reference to `sqlite3_finalize'
examples/hw: In function `pysqlite_statement_finalize':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:355: undefined reference to `sqlite3_finalize'
examples/hw: In function `pysqlite_statement_reset':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/statement.c:373: undefined reference to `sqlite3_reset'
examples/hw: In function `pysqlite_cursor_executescript':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:799: undefined reference to `sqlite3_prepare'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:826: undefined reference to `sqlite3_finalize'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:815: undefined reference to `sqlite3_finalize'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:821: undefined reference to `sqlite3_finalize'
examples/hw: In function `pysqlite_build_row_cast_map':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:176: undefined reference to `sqlite3_column_count'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:180: undefined reference to `sqlite3_column_name'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:203: undefined reference to `sqlite3_column_decltype'
examples/hw: In function `_pysqlite_fetch_one_row':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:288: undefined reference to `sqlite3_data_count'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:322: undefined reference to `sqlite3_column_type'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:371: undefined reference to `sqlite3_column_bytes'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:373: undefined reference to `sqlite3_column_blob'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:306: undefined reference to `sqlite3_column_bytes'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:307: undefined reference to `sqlite3_column_blob'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:342: undefined reference to `sqlite3_column_name'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:332: undefined reference to `sqlite3_column_text'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:333: undefined reference to `sqlite3_column_bytes'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:328: undefined reference to `sqlite3_column_int64'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:330: undefined reference to `sqlite3_column_double'
examples/hw: In function `_pysqlite_query_execute':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:732: undefined reference to `sqlite3_get_autocommit'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:705: undefined reference to `sqlite3_changes'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:658: undefined reference to `sqlite3_column_count'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:671: undefined reference to `sqlite3_column_name'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:732: undefined reference to `sqlite3_get_autocommit'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:705: undefined reference to `sqlite3_changes'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:712: undefined reference to `sqlite3_last_insert_rowid'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:658: undefined reference to `sqlite3_column_count'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:671: undefined reference to `sqlite3_column_name'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:732: undefined reference to `sqlite3_get_autocommit'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:705: undefined reference to `sqlite3_changes'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:712: undefined reference to `sqlite3_last_insert_rowid'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:658: undefined reference to `sqlite3_column_count'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/cursor.c:671: undefined reference to `sqlite3_column_name'
examples/hw: In function `pysqlite_step':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:37: undefined reference to `sqlite3_step'
examples/hw: In function `_pysqlite_seterror':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:54: undefined reference to `sqlite3_reset'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:57: undefined reference to `sqlite3_errcode'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:100: undefined reference to `sqlite3_errmsg'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:94: undefined reference to `sqlite3_errmsg'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:97: undefined reference to `sqlite3_errmsg'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:84: undefined reference to `sqlite3_errmsg'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:66: undefined reference to `sqlite3_errmsg'
examples/hw:/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/util.c:90: more undefined references to `sqlite3_errmsg' follow
examples/hw: In function `pysqlite_connection_dealloc':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:253: undefined reference to `sqlite3_close'
examples/hw: In function `_pysqlite_set_result':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:537: undefined reference to `sqlite3_result_blob'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:518: undefined reference to `sqlite3_result_double'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:516: undefined reference to `sqlite3_result_int64'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:523: undefined reference to `sqlite3_result_text'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:511: undefined reference to `sqlite3_result_null'
examples/hw: In function `_pysqlite_final_callback':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:720: undefined reference to `sqlite3_aggregate_context'
examples/hw: In function `_sqlite3_result_error':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:56: undefined reference to `sqlite3_result_error'
examples/hw: In function `pysqlite_connection_interrupt':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1447: undefined reference to `sqlite3_interrupt'
examples/hw: In function `pysqlite_connection_get_total_changes':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1171: undefined reference to `sqlite3_total_changes'
examples/hw: In function `pysqlite_enable_load_extension':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1107: undefined reference to `sqlite3_enable_load_extension'
examples/hw: In function `pysqlite_load_extension':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1132: undefined reference to `sqlite3_load_extension'
examples/hw: In function `pysqlite_connection_set_trace_callback':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1086: undefined reference to `sqlite3_trace'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1082: undefined reference to `sqlite3_trace'
examples/hw: In function `pysqlite_connection_set_authorizer':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1022: undefined reference to `sqlite3_set_authorizer'
examples/hw: In function `pysqlite_connection_set_progress_handler':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1056: undefined reference to `sqlite3_progress_handler'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1054: undefined reference to `sqlite3_progress_handler'
examples/hw: In function `pysqlite_connection_rollback':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:475: undefined reference to `sqlite3_prepare'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:490: undefined reference to `sqlite3_finalize'
examples/hw: In function `pysqlite_connection_create_function':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:853: undefined reference to `sqlite3_create_function'
examples/hw: In function `pysqlite_connection_create_aggregate':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:886: undefined reference to `sqlite3_create_function'
examples/hw: In function `pysqlite_connection_commit':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:429: undefined reference to `sqlite3_prepare'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:444: undefined reference to `sqlite3_finalize'
examples/hw: In function `pysqlite_connection_init':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:104: undefined reference to `sqlite3_open_v2'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:163: undefined reference to `sqlite3_busy_timeout'
examples/hw: In function `pysqlite_connection_create_collation':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:1564: undefined reference to `sqlite3_create_collation'
examples/hw: In function `pysqlite_connection_close':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:341: undefined reference to `sqlite3_close'
examples/hw: In function `_pysqlite_connection_begin':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:383: undefined reference to `sqlite3_prepare'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:399: undefined reference to `sqlite3_finalize'
examples/hw: In function `_pysqlite_build_py_params':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:579: undefined reference to `sqlite3_value_bytes'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:581: undefined reference to `sqlite3_value_blob'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:561: undefined reference to `sqlite3_value_type'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:563: undefined reference to `sqlite3_value_int64'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:566: undefined reference to `sqlite3_value_double'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:569: undefined reference to `sqlite3_value_text'
examples/hw: In function `_pysqlite_func_callback':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:614: undefined reference to `sqlite3_user_data'
examples/hw: In function `_sqlite3_result_error':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:56: undefined reference to `sqlite3_result_error'
examples/hw: In function `_pysqlite_step_callback':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:655: undefined reference to `sqlite3_user_data'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:657: undefined reference to `sqlite3_aggregate_context'
examples/hw: In function `_sqlite3_result_error':
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:56: undefined reference to `sqlite3_result_error'
/home/vagrant/rumprun-packages/python/build/./Modules/_sqlite/connection.c:56: undefined reference to `sqlite3_result_error'
collect2: error: ld returned 1 exit status
vagrant@vagrant-xen-trusty64:~/rumprun-packages/python$
</pre>

How can I get rid of this error? Do I need to rebuild the ``python`` package ?

--- a/python/README.md
+++ b/python/README.md
@@ -72,8 +72,11 @@ may see some link errors related to rc5. This doesn't affect the example.
 x86_64-rumprun-netbsd-gcc examples/hw.c \
 	-o examples/hw \
 	-Ibuild/pythondist/include/python3.4m \
+	-I../pkgs/include \
 	-Lbuild/pythondist/lib \
-	-lpython3.4m -lutil -lm -lz -lssl -lcrypto
+	-L../pkgs/lib \
+
+	-lpython3.4m -lutil -lm -lz -lssl -lcrypto -lsqlite3
 ```
 
 Now you've got a unikernel image. You just need to bake it. We'll use the hw_generic since

